config/initializers/clearance.rb

```diff
@@ -1,3 +1,5 @@
+require_relative "../../lib/confirmed_user_guard"
```

Comment:
@simi do you have context on the need for this change? require_relative is not a huge issue but constant missing/module undefined keeps happening with every rails update.

Reply:
The issue for us was in production env. docker build was failing on asset precompile:
```
#23 [build 12/13] RUN RAILS_ENV=production RAILS_GROUPS=assets SECRET_KEY_BASE=1234 bin/rails assets:precompile
#23 4.651 rails aborted!
#23 4.651 NameError: uninitialized constant Lograge::Formatters::Datadog
#23 4.651 /app/config/initializers/lograge.rb:7:in `block in <main>'
```
https://github.com/rubygems/rubygems.org/runs/4872242431?check_suite_focus=true#step:6:512

I was wondering if we could identify the real issue, we could use the recommended fix. I guess require_relative will do for now. We have use same fix all the previous times it broke https://github.com/rubygems/rubygems.org/pull/2598